test/legacy.js

```diff
@@ -28,11 +28,12 @@ const botA = createSbot({
   keys: alice
 })
 
-tape(function (t) {
-  t.throws(function () {
-    botA.ebt.replicate.call(bob, { version: 1 })
-  })
+tape('legacy', function (t) {
+  setTimeout(() => {
+    t.throws(function () {
+      botA.ebt.replicate.call(bob, { version: 1 })
+    })
 
-  botA.close()
-  t.end()
+    botA.close(t.end)
+  }, 100)
```

Comment:
why was the timeout added here?

Reply:
I suspect it's a race condition with the multiserver inside the sbot not having time to start, and then closing the sbot before it's allowed. Because without the setTimeout, I consistently get this error:

```
not ok 2 Error [ERR_SERVER_NOT_RUNNING]: Server is not running.
  ---
    operator: error
    at: <anonymous> (/home/staltz/oss/ssb-ebt/node_modules/secret-stack/lib/core.js:261:31)
    stack: |-
      Error [ERR_SERVER_NOT_RUNNING]: Server is not running.
          at Server.close (net.js:1605:12)
          at Object.onceWrapper (events.js:420:28)
          at Server.emit (events.js:314:20)
          at emitCloseNT (net.js:1658:8)
          at processTicksAndRejections (internal/process/task_queues.js:83:21)
```

Reply:
Hmm, actually this timeout is also not *consistently* fixing the error. :disappointed: